web/src/compose_closed_ui.ts

```diff
@@ -83,9 +90,36 @@ function update_reply_button_state(disable = false): void {
 }
 
 function update_buttons(disable_reply?: boolean): void {
+    if (disable_reply && maybe_get_selected_message_stream_id()) {
+        $("#new_conversation_button").prop("disabled", true);
+    } else {
+        $("#new_conversation_button").prop("disabled", false);
+    }
```

Comment:
it's better to extract it to a separate function like `update_conversation_button_state` just like `update_reply_button_state`.

Reply:
That sounds good. Named the function - `update_new_conversation_button_state`.